actgen/agents/__init__.py

```diff
@@ -1 +1,12 @@
-from .randomagent import RandomAgent
\ No newline at end of file
+from .randomagent import RandomAgent
+from .dqnagent import DQNAgent
+
+
+def save_model(model, path):
+    # TODO:
+    pass
```

Comment:
I have code for saving and loading already implemented that we can use

Reply:
oh awesome. I wrote these functions a while ago and totally forgot to implement them lol

Reply:
Sync with the main branch and have a look at nnutils.py. When making a new module, like your `DQN`, I usually subclass `Network` (rather than `torch.nn.Module`) so I get saving/loading, freezing/unfreezing, and parameter counting for free.

Reply:
I'm not married to the name `nnutils`, but it feels separate enough from the other utils that I wanted it in a different file.